<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IniFile manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>assets/create_icon.png</normaloff>assets/create_icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>assets/open_icon.png</normaloff>assets/open_icon.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>assets/save_icon.png</normaloff>assets/save_icon.png</iconset>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="settingsSection">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>531</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>251</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Editing</string>
     </property>
     <widget class="QLineEdit" name="keyEditLine">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>63</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Key</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="valueEditLine">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>63</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>61</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Delete</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>63</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Key</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>138</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QComboBox" name="deleteComb">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sections</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sectionsComb">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>301</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sectionAdd">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>51</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sectionDelete">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>61</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current file:</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentFileLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>9</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>No file opened yet</string>
    </property>
   </widget>
   <widget class="QListWidget" name="fileListWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>51</y>
      <width>451</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>51</y>
      <width>531</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate_new"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionSave_all"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate_new">
   <property name="text">
    <string>Create new</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionSave_all">
   <property name="text">
    <string>Save all</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
